### ISSUE TITLE
Single Patient API: 10.10.08 All must support elements are provided in the DiagnosticReport resources returned

### ISSUE THREAD

**Issue by VighneshAlevoorP** [onc-healthit/onc-certification-g10-test-kit#593]
We are seeing one issue as - Could not find result in the 4 provided DiagnosticReport resource(s). But in our response we are giving result as shown below. Please confirm what changes are needed for this.

<img width="686" alt="image" src="https://github.com/user-attachments/assets/9e07ad80-e5ad-4d0b-9a62-8d2ab23a9054">

We have the response as below but still it is getting skipped.
<img width="392" alt="image" src="https://github.com/user-attachments/assets/e7bf32b0-0db0-4f7d-9e02-3ff1ff42edfe">

**Comment by yunwwang** [comment]
That is strange. Did you run on inferno.healthit.gov? If so, could you send me the test session url?

**Comment by VighneshAlevoorP** [comment]
Yes,  URL is-  https://inferno.healthit.gov/suites/g10_certification/5tHOxGhh63a#10

**Comment by yunwwang** [comment]
OK. This is about which reosurces the MustSupport test inspects. The MustSupport only inspects the resources returned in the "first" search. For DiagnoasticReport for Report and Note Exchange test group, the first search is search by category + patient (test 10.10.01). In that test, sever returns two DiagnosticReport resources but none of them have `result` element. Though server returns additional resources in the following search by patient test (10.10.02) but those additional resources are not "DiagnoasticReport for Report and Note Exchange" resources. So the server has not demonstrated that it support `result` element for DiagnoasticReport for Report and Note Exchange profile specifically.

**Comment by VighneshAlevoorP** [comment]
We still see the issue here. 
https://inferno.healthit.gov/suites/g10_certification/3qwVxcKhm8X#10

In 10.10.01 - Server returns valid results for DiagnosticReport search by patient + category
It is making below call, I can see result in our postman but Inferno response I do not see any data related to result.

https://ap25sandbox.fhirapi.athenahealth.com/demoAPIServer/fhir/r4/DiagnosticReport/_search

<img width="637" alt="Image" src="https://github.com/user-attachments/assets/513a0a91-edd7-4092-afef-a8896a474e97" />

**Comment by yunwwang** [comment]
@VighneshAlevoorP I check the result in your test session, I don't see any DiagnosticReport resource having result element. It could be I missed it since there are multiple calls. Can you point me to the resource(s) in your test session that having result element? I noticed the bearer token used in postman is different than the one used in test session. That could cause different resources returned.

**Comment by VighneshAlevoorP** [comment]
@yunwwang Thanks for the info. Closing the ticket.
